activesupport/lib/active_support/cache.rb

```diff
@@ -85,6 +85,14 @@ def expand_cache_key(key, namespace = nil)
         expanded_cache_key
       end
 
+      def expand_cache_version(key)
```

Comment:
This method is currently public. I am not sure why would anyone call it explicitly. I would make it private. Otherwise it needs to be documented.